config/default.json

```diff
@@ -1,7 +1,7 @@
 {
     "SECRET_KEY": "ahhhhhhhhhh",
 
-    "MYSQL_HOST": "127.0.0.1",
+    "MYSQL_HOST": "mysql",
```

Comment:
This also assumes a Docker setup, which while is the default, doesn't mean we should break other dev environments. I think the best way to handle this would be to have a config/docker.json with this Docker setup, and then set some environment variable in docker or something to signal the application to use that config.